nubilanding/src/components/calculadora/calculadora.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styles from "../calculadora/styles.module.scss";
+import Flecha from "../../assets/ic_calculator_arrow.svg";
+
+class Calculadora extends React.Component {
+  render() {
+    return (
+      <div className={styles.contenedor}>
+        <div>
+          <h2 className={styles.titulo}>Calculá tu retiro o recarga</h2>
+        </div>
+        <form className={styles.calculadora}>
+          <div className={styles.seccionValor}>
+            <input type="text" placeholder="0" name="monto" id="monto" />
+            <label>Indicá el monto en Dólares</label>
+            <label className={styles.inputRadio} htmlFor="recarga">RECARGA
+              <input type="radio" id="recarga" name="tipodeoperacion" value="recarga"/>
+              <span className={styles.marcado}></span>
+            </label>
```

Comment:
Revisa la identación acá y recordá que los elementos vacíos se pueden cerrar en el primer tag:

```suggestion
            <label className={styles.inputRadio} htmlFor="recarga">
              RECARGA
              <input type="radio" id="recarga" name="tipodeoperacion" value="recarga"/>
              <span className={styles.marcado} />
            </label>
```